mlflow/langchain/__init__.py

```diff
@@ -151,6 +158,13 @@ def save_model(
 
                      .. Note:: Experimental: This parameter may change or be removed in a future
                                              release without warning.
+    :param loader_fn: A function that takes a string `persist_dir` defined below as the argument
+                      and returns a retriever. This is required for RetrievalQA models.
```

Comment:
Can we include some example code for this?

Reply:
Below all of the parameter definitions, can we include a somewhat-complete example of logging a RetrievalQA chain with `loader_fn` and `persist_dir`?

Reply:
An example is provided in `examples/langchain/retrieval_qa_chain.py`. Do we want to mention this file in this docstring?

Reply:
Yes, we definitely want it in the docstring. Users typically look at API docs first. The examples GitHub repo isn't sufficient

Reply:
Updated